### PR TITLE
create_activities

### PR DESCRIPTION
**Pull request by matthewrsj**
ref #31 and #33

### CODE REVIEW

**Comment by Kennric** [comment]
Tests look good. +1

File: tests.py [hunk 1]
```diff
@@ -374,15 +367,146 @@ def test_post_project_type_error(self):
...
+        # Instantiate timesync class
+        ts = pymesync.TimeSync(baseurl,
+                               password="password",
+                               user="example-user",
+                               auth_type="password")
+
+        for key in params:
+            del(params_to_test[key])
+            self.assertEquals(ts.create_activity(params_to_test), [{
+                "pymesync error":
```
Comment (athai) [inline tests.py]: Is the change to double quotes intentional?

Comment (matthewrsj) [inline tests.py]: No, just inconsistency on my part. You are right, I should be consistent.

**Comment by Kennric** [comment]
Looks good to me. +1

**Pull request merged**